website/static/js/project-organizer.js

```diff
@@ -364,10 +375,14 @@ var tbOptions = {
                     }
                     tb.filter(event);
                 },
-                onchange: m.withAttr('value', tb.filterText),
+                onchange: function() {
+                    m.withAttr('value', tb.filterText);
```

Comment:
Does this still work, i.e. is `filterText` still properly updated? As far as I understand, `m.withAttr` returns *a function*, which is why 

```js
onchange: m.withAttr('value', tb.filterText)
```

works. But now, you are explicitly declaring a callback function here, so I don't think this line is doing anything. I would expect you'd need to do

```js
onchange: function(e) {
    tb.filterText(e.value);
    // ...
}
```

though you should verify this.